src/server.rs

```diff
@@ -510,13 +509,20 @@ impl<S : CallStarter> httpbis::Service for GrpcHttpService<S> {
                             ))
                     }
                 })
+                .normalize_metadata(
```

Comment:
I think it is safer here to assume that `grpc_response` stream may not contain trailing metadata. And if it does not, then HTTP response stream won't contain mandatory trailing headers.